### PR TITLE
Form validate

### PR DESCRIPTION
**Pull request by mstanley-me**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+hamburger.addEventListener('click', () => {
+    hamburger.classList.toggle('active', navItems);
+    mobileMenu.classList.toggle('active', navItems);
+});
+
+
+document.querySelectorAll('.link').forEach((event) => event.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    mobileMenu.classList.remove('active');
+}));
```
Comment (oloomoses) [inline script.js:14]: - [ ] You have done very well fixing the errors raised in the previous reviews. However, clicking the close(X) button does not close the window. Kindly ensure that When the user clicks (or taps) the close (X) button, the mobile menu disappears.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @mstanley-me, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (mstanley-me) [reply script.js]: That's good now!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const navItems = document.querySelector('.nav-items');
+const openMenu = document.querySelector('.openMenu');
+const closeMenu = document.querySelector('.closeMenu');
+
+
+openMenu.addEventListener('click', show);
+closeMenu.addEventListener('click', close);
```
Comment (mateo951) [inline script.js:7]: Addressed 🔥 The menu closes as expected when clicking the X button✔

**Review by mateo951 — APPROVED** [review]
### Hi @mstanley-me, 

Well done implementing the changes requested by the previous reviewer 👌
Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mstanley-me** [comment]
Thank you!

**Pull request merged**